components/licensor/ee/pkg/licensor/replicated.go

```diff
@@ -118,6 +118,9 @@ func newReplicatedEvaluator(client *http.Client) (res *Evaluator) {
 
 		case "seats":
 			lic.Seats = int(i.Value.(float64))
+
+		case "customerId":
```

Comment:
There are two hard problems in computer science; naming, cache invalidation, and off by one errors.

Within Replicated we have a custom `levelId` field; at present I'm naming this field `customerId` to match that pattern. However within this implementation we use `customerID` with an upper cased D. If anyone has strong opinions I'm happy to adjust this.